### DISCUSSION TITLE
테스트 링크에 접속해 인앱결제 시 실제 젬이 결제/차감됩니다

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#599] [Issue Report (Legacy)]
테스터로 등록한지 2주 좀 넘게 지났음에도
(테스터가 아니면 접속자체가 안되지만..)
IWP 버튼을 통해 인앱결제시 실제 젬이 결제됩니다.

서비스 중이지 않은 월드인데(심사 전) 정상적인 상황이 아니라고 보여져서 이슈 올립니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 해당 이슈 결제 로그 확인을 위해 월드 id를 알려주시기를 부탁드립니다.
